### ISSUE TITLE
Program Hang opening log file over network

### ISSUE THREAD

**Issue by GoogleCodeExporter** [snakefoot/snaketail-net#46]
```
What steps will reproduce the problem?
1. Open Snake Tail
2. Navigate to \\computer\c$\path\to\file
3. Snake Tail hangs

What is the expected output? What do you see instead?
Complete app hang


What version of the product are you using? On what operating system?
1.8.2.0 on Windows 8 Enterprise 64 bit


Please provide any additional information below.
Log file in question was a robocopy log.


```

Original issue reported on code.google.com by `baker.matt.b` on 9 Jul 2013 at 6:47

**Comment by zspitzer** [comment]
I am just trying this tool out, I also got a not responding opening a 5mb file from a network drive

**Comment by snakefoot** [comment]
@zspitzer I have a few questions:
How long does it take for Notepad to open the file?
Can you try and create a memory-dump of SnakeTail, when its blocking? (Using Task Manager or rather ProcDump)

**Comment by zspitzer** [comment]
Notepad takes two or three seconds, SnakeTail hangs for minutes

How should i send you the procdump file, it's 3.5mb zipped

**Comment by snakefoot** [comment]
Closed with 80e5ee624dc6f0184e81c19f9f6d2476a38ae613 (Version 1.9.2)
